digital_comms/ccam.py

```diff
@@ -160,7 +271,7 @@ def coverage(self):
 
 
 class PostcodeSector(object):
-    """Represents a pcd_sector to be modelled
+    """""Represents a Postcode sector to be modelled
```

Comment:
Have we got the doc string wrong here, shouldn't it be three double quotes (""")?

Reply:
Done